src/main/java/com/google/api/gax/grpc/PagedListResponseImpl.java

```diff
@@ -50,7 +49,7 @@
   /** */
   public PagedListResponseImpl(
       RequestT request,
-      FutureCallable<RequestT, ResponseT> callable,
+      UnaryApiCallable<RequestT, ResponseT> callable,
       PageStreamingDescriptor<RequestT, ResponseT, ResourceT> pageDescriptor,
       CallContext context) {
```

Comment:
Could we change the order of these arguments while we're changing stuff here? I recommend: callable, pageDescriptor, request, context.